addons/binding/org.openhab.binding.unifi/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,31 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: UniFi Binding
+Bundle-SymbolicName: org.openhab.binding.unifi;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.3.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: 
+ com.fasterxml.jackson.annotation,
+ com.fasterxml.jackson.core,
+ com.fasterxml.jackson.databind,
+ com.fasterxml.jackson.databind.annotation,
+ com.google.common.collect,
```

Comment:
You can remove the dependency on `com.google.common.collect` when you replace the set logic with Java 8 code like:

```
    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Stream
            .of(UniFiBindingConstants.THING_TYPE_CONTROLLER, UniFiBindingConstants.THING_TYPE_CLIENT)
            .collect(Collectors.toSet());
```